dex/testing/btc/base-harness.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/sh
+# Tmux script that sets up a simnet harness.
+set -e
+set -x # for verbose output
```

Comment:
`set -ex` does the trick too, but no objection if this reads better with the comment.